idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -383,17 +370,18 @@ def _make_performance(self):
         """
 
         # ======================================================================
-        # Aliases for  Sets
+        # Custom Sets
         vap_comp = self.config.vapor_side.property_package.component_list
         liq_comp = self.config.liquid_side.property_package.component_list
-        dcomp = self.config.liquid_side.property_package.component_list_d
-        vapor_phase_list_ref = self.config.vapor_side.property_package.phase_list
-        liquid_phase_list_ref = self.config.liquid_side.property_package.phase_list
-
-        # Add object reference - time
-        add_object_reference(self,
-                             "t",
-                             self.flowsheet().time)
+        equilibrium_comp = vap_comp & liq_comp
+        solvent_comp_list = \
+            self.config.liquid_side.property_package.solvent_set
+        solute_comp_list = self.config.liquid_side.property_package.solute_set
```

Comment:
We need to think about this more - either we need to make it a requirement that the liquids phase property package defines solvents and solutes (and that we add a check for this), or we need to generalize the model more.

My initial feeling is that we do not need this, at least at the general column level, and that we just need to make the assumption that any component that appears in both phases is in equilibrium, and any that appears in only one phase is non-condensable/non-vapourisable.

Again, see the Solvent Reboiler model for an example.